src/OpenSage.Game/Gui/ControlBar/CommandButton.cs

```diff
@@ -8,6 +8,15 @@ namespace OpenSage.Gui.ControlBar
 {
     public sealed class CommandButton : BaseAsset
     {
+        public CommandButton(){ }
```

Comment:
I don't think this is valid OpenSage formatting

Reply:
done